web/static/ts/utils/dom.ts

```diff
@@ -29,3 +29,28 @@ export function insertAfter(elem: Element, refElem: Element): Element {
 
     return refElem.parentElement.insertBefore(elem, refElem.nextElementSibling);
 }
+
+/**
+ * @param alert The alert caused by the User/System.
+ */
+export function alertBox(alert: alertMessage) {
+    const error = document.querySelector("#errorAlert");
+    const errorBtn = document.querySelector("#errorAlertButton");
```

Comment:
These ID names don't follow the naming convention. The convention would have them named `error-alert` etc.